CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 4.0.0 - TBD
 
 ### Changed
+- **BREAKING**: Uses `v1.driver(options)` API instead of `v1.veres()`
```

Comment:
I think this needs punctuation (a period) at the end.

Reply:
Thanks, fixed.